<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVLcaLccResultsWidget</class>
 <widget class="QWidget" name="SVLcaLccResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LCA/LCC Results</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabWidgetPageLCA">
      <attribute name="title">
       <string>Life cycle assessment results</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidgetLcaResults">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPageLCC">
      <attribute name="title">
       <string>Life cycle cost results</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidgetLccResults"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidgetLccOverview">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
